docker/run-fmt

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker run -v `pwd`:/holochain --rm -p 3141:3141 -it holochain/rust-ubuntu:latest cargo fmt
```

Comment:
@sphinxc0re FYI it's `holochain/rust-ubuntu:develop` as we don't have a `:latest` tag atm, i'll update and push